coderd/notifications/notifier.go

```diff
@@ -291,7 +298,7 @@ func (n *notifier) newSuccessfulDispatch(msg database.AcquireNotificationMessage
 	return dispatchResult{
 		notifier: n.id,
 		msg:      msg.ID,
-		ts:       dbtime.Now(),
+		ts:       dbtime.Time(n.clock.Now()),
```

Comment:
This introduces a subtle behaviour change because unfortunately `dbtime.Now` forces the input to UTC while `dbtime.Time` does not.

```suggestion
		ts:       dbtime.Time(n.clock.Now().UTC()),
```